tools/dump_tool/ramdumpParser.py

```diff
@@ -563,6 +563,17 @@ def main():
 	print('*************************************************************')
 	print('')
 
+	# Read config information
+	fd = open(config_path, 'r')
+	data = fd.read()
+	fd.close()
+
+	# Get arch family
+	if ('CONFIG_ARCH_FAMILY="armv8-m"' in data) or ('CONFIG_ARCH_FAMILY="armv7-m"' in data):
```

Comment:
The ramdump parser tool is basically used to get the dump call stack as well as other information like heap, etc.. 
Our logic here is related to printing the call stack symbols in case of cortex m, where the FP is not available. So, I think we must move this logic to line 621. 

`if cortex m`
`then do our logic`
`else`
`do existing call stack logic using FP`
`...`
`continue with remaining ramdump parser logic for heap, etc.. `